migrator/readme.md

```diff
@@ -0,0 +1,49 @@
+## Running a migration
+
+The migrator sub project produces a executable jar which will migrate data from an R2 database into the path manager.
+
+To run you will need a `migrator.properties` file in the same directory you are executing from. The format of this file is:
+
+```
+    databaseAddress=10.0.0.127
+    databaseService=gudevelopersdb.gucode.gnl
+    user=deliveryXX 
+    password=XXXXXXXXXXXX 
+    pathManagerUrl=http://pathmanager.local.dev-gutools.co.uk/
```

Comment:
nit: this is a dev-nginx address. dev-nginx will setup a redirect from http tp https. This value could be https to remove that hop.

Reply:
Could well be, I don't really want to update the migrator docs as this move is a weak deprecation.